vanilla-framework/vanilla-dashboard-theme
# Framework cleanup

## Done

- All SVGs have been optimised via SVGO and now embedded within the framework, removing the reliance of the asset server.
- Added improvements to the icon files, added the ability for the spinner to work on any background and removes the requirement for a different class.
- Renamed files and mixins to follow the VF naming principles.


## QA

- Pull down the branch, do `./run`
- Check locally all files load and all icons load too
